### DISCUSSION TITLE
How to use the prebuilt GraphHopper files?

### DISCUSSION THREAD

**Discussion by dingxmhan** [mapsforge/mapsforge#1564] [Q&A] [answered]
thank you very much at first. is there any document to teach us how to use the Prebuilt Graphhopper Files at https://download.mapsforge.org/graphs/ to do navigation？

thank you very much.

**Comment by devemux86** [comment, marked as answer]
GraphHopper no longer seems to support offline routing:
- https://github.com/graphhopper/graphhopper/issues/1940

There is an old [Android sample](https://github.com/graphhopper/graphhopper/tree/1.0/android).

---

For offline routing better check **[BRouter](https://github.com/abrensch/brouter/)**.
